refactor : 코드 포맷을 맞추고, 테스트코드 메서드를 한글로 변경하라

bloomspes opened the pull request:
### 1. 코드 포맷팅을 했습니다.

(MacOS 기준)
- `cmd + option`, `L`
- `control + option`, `O`  
- 두 단축키를 습관화 합시다.

### 2. ProductControllerTest 의 테스트코드 메서드를 한글로 변경했습니다.
<img width="738" alt="Screen Shot 2021-10-14 at 5 25 45 PM" src="https://user-images.githubusercontent.com/27952469/137279943-6c005236-b0a9-4f4e-973b-5c3c5f2591b5.png">
.

bloomspes committed:
refactor : 코드 포맷을 맞추고, 테스트코드 메서드를 한글로 변경하라

Commit:
Google Java Format

mikekang47 commented on src/test/java/com/dddkurlyclone/product/controllers/ProductControllerTest.java:
```diff
@@ -37,19 +37,19 @@ void setUp() {
 
   @Test
-  void create() throws Exception {
+  void 상품_데이터를_생성하는가() throws Exception {
```
메서드 명이 Test와 잘 어우러 지는 것 같아요!

mikekang47 reviewed (commented):
한글 메서드가 DisplayName을 사용할 필요도 없고 한 눈에 들어오네요! 수고하셨습니다👍

juunini commented on src/test/java/com/dddkurlyclone/product/controllers/ProductControllerTest.java:
```diff
@@ -27,7 +27,7 @@
 
   @BeforeEach
-  void setUp() {
+  void 테스트를_준비_합니다() {
     Product product = Product.builder().id(1L).name("샤인마토").price(7000).build();
 
     given(productService.getProducts()).willReturn(List.of(product));
```
이거 뭐랄까... 얘는 setup의 역할을 하는 코드가 아닌 것 같아요.

bloomspes replied:
오잉 어째서죠..?

juunini commented on src/test/java/com/dddkurlyclone/product/controllers/ProductControllerTest.java:
```diff
@@ -37,19 +37,19 @@ void setUp() {
 
   @Test
-  void list() throws Exception {
+  void 상품_목록을_반환하는가() throws Exception {
```
'반환' 대신 '응답' 은 어떠신가요?

bloomspes replied:
응답이라는 어휘를 쓰려면.. 상품 목록 요청에 응답하는가 이렇게 적는 것을 의미할까요?

sunggyupaik reviewed (commented):
고생하셨습니다~! 저희 코드 포맷이 좀 많이 지저분헀나보네요 ㅎㅎ 엄청 나게 많은 수정이 ㅋㅋㅋㅋ

bloomspes commented on src/test/java/com/dddkurlyclone/product/controllers/ProductControllerTest.java:
```diff
@@ -20,41 +11,53 @@
+    }
+
+    @Test
+    void 상품_목록을_반환하는가() throws Exception {
```
맥락상 `상품 목록` 이라는 표현이 맞습니다. :) 개별 아이디로 출력되는 것은 특정 상품만 거론되는것이 아니라, 상품이 가지고 있는 디테일한 정보도 포함이 되어서 그렇습니다.

bloomspes committed:
refactor : 코드 라인을 맞추고, 테스트케이스의 메소드를 변경하라.

- 코드 라인 포맷팅
- 테스트케이스의 메소드명을 반환합니다 -> 요청에 응답하는가 로 수정

bloomspes committed:
refactor : 코드 라인 포맷팅, 테스트 메서드명 수정

Merge branch 'test-refactor' of https://github.com/bloomspes/DDD-Kurly-Clone-Product into test-refactor

Commit:
Google Java Format

bloomspes commented:
요구사항을 반영했습니닷!
